docker/complement/conf/workers-shared-extra.yaml.j2

```diff
@@ -92,3 +94,5 @@ server_notices:
   system_mxid_display_name: "Server Alert"
   system_mxid_avatar_url: ""
   room_name: "Server Alert"
+
+{% include "shared-orig.yaml.j2" %}
```

Comment:
I'll add comments.
As for why this is only now necessary: Faster Room Joins were already disabled for workers before. Since merging the two configuration files into one, I need to be able to choose that dynamically.
This means this part of the configuration file needs to be written in Jinja2 (or at least, I'd prefer that to alternatives).

Originally, this file gets copied to `/conf/workers/shared.yaml`. What really wasn't obvious to me is that `convert` then ***appends*** the result of a Jinja2 template (`/conf/shared.yaml.j2`) to that file. The comment about 'Replacing' the file seems to have been bogus.

As steps:

1. workers Dockerfile: Create template file `/conf/shared.yaml.j2`.
2. Complement-workers Dockerfile: Create base file `/conf/workers/shared.yaml`. 
3. Configuration script: Append `jinja2(/conf/shared.yaml.j2 [1])` to `/conf/workers/shared.yaml`

What *now* happens is:

1. workers Dockerfile: Create template file `/conf/shared.yaml.j2`
2. Complement Dockerfile: Move [1] to `/conf/shared-orig.yaml.j2`.
3. Complement Dockerfile: Create template file `/conf/shared.yaml.j2`.
4. Configuration script: Write `jinja2(/conf/shared.yaml.j2 [Complement, 3])` to `/conf/workers/shared.yaml`. (This includes `/conf/shared-orig.yaml.j2 [workers, 1-2]` through a Jinja2 directive.)

The only semantic difference is that we now can use Jinja2 in the 'base' template.